apps/api/src/api/application-api.ts

```diff
@@ -116,3 +117,51 @@ export const getApplicationStateTotals = async ({ userId }: { userId: string })
 
 	return await service.applicationStateTotals({ user_id: userId });
 };
+/**
+ * Description placeholder
```

Comment:
This comment looks unfinished? I see differences between Git and the branch locally so I'm bit unsure what's going on there